cmd.js

```diff
@@ -72,7 +72,11 @@ function runner (command) {
     if (running) return
     running = true
     verboseLog('Executing command: ' + command)
-    execAsync(command, function (err) {
+
+    var env = process.env
+    env.FILENAME = path
+    env.EVENT = event
```

Comment:
I'd create a new object from `process.env` with `Object.assign({}, process.env, { FILENAME: path, EVENT: event})`. Modifying `process.env` directly also modifies the environment of the running process and I'd rather do that without a reason. As this package still supports node 0.12 the `Object.assign` function has to be imported from the `object-assign` package.